src/Rings/Laurent.jl

```diff
@@ -241,3 +241,13 @@ end
 
 
 
+function quo(R::Oscar.LaurentMPolyWrapRing, I::Oscar.LaurentMPolyIdeal)
+  poly_repr = Oscar._polyringquo(R);
```

Comment:
```suggestion
  poly_repr = Oscar._polyringquo(R)
```
please remove all semicolons that are not needed (so basically all)

Reply:
Something to do at the very end when switching to a non-draft...  After any "semantic" issues have been dealt with.